content/docs/setup/kubernetes/quick-start-alicloud-ack/index.md

```diff
@@ -0,0 +1,82 @@
+## Setup
+
+### Deploy Istio through Application Catalog
+
+-  Log on to the AliCloud Container Service console, and click `Application Catalog` in the left navigation pane, then select `ack-istio` in the right panel.
+
+{{< image width="100%" ratio="67.17%"
+    link="./app-catalog-istio-1.0.0.png"
+    caption="Istio"
+    >}}
+
+-  Customization with Parameters
+
+The Helm chart ships with reasonable default configuration options which are explained in the following table:
+
+| Parameter                            | Description                                                  | Default                                    |
+| ------------------------------------ | ------------------------------------------------------------ | ------------------------------------------ |
+| `global.hub` | Specifies the HUB for most images used by Istio | registry.cn-hangzhou.aliyuncs.com/aliacs-app-catalog |
+| `global.tag`                     | Specifies the TAG for most images used by Istio |    0.8       |
+| `global.proxy.image`             | Specifies the proxy image name         | istio-proxyv2         |
+| `global.imagePullPolicy`       | Specifies the image pull policy          | `IfNotPresent`        |
+| `global.controlPlaneSecurityEnabled` | Specifies whether control plane mTLS is enabled | `false` |
+| `global.mtls.enabled`        | Specifies whether mTLS is enabled by default between services| `false`  |
+| `global.mtls.mtlsExcludedServices`  | List of FQDNs to exclude from mTLS | -"kubernetes.default.svc.cluster.local" |
+| `global.rbacEnabled` | Specifies whether to create Istio RBAC rules or not | `true` |
+| `global.refreshInterval` | Specifies the mesh discovery refresh interval | `10s` |
+| `global.arch.amd64` | Specifies the scheduling policy for amd64 architectures | `2` |
+| `global.arch.s390x` | Specifies the scheduling policy for s390x architectures | `2` |
+| `global.arch.ppc64le` | Specifies the scheduling policy for ppc64le architectures| `2` |
+| `galley.enabled` | Specifies whether Galley should be installed for server-side config validation. Requires k8s >= 1.9 | `false` |
+
+The Helm chart also offers significant customization options per individual service. Customize these per-service options at your own risk. The per-service options are exposed via the Parameters tab.
+
+Wait until Istio is fully deployed. Note that this can take up to several minutes.
+
+## What's next
+
+You can further explore the Istio functionality by following any of the tutorials in the [Guides](/docs/guides/) section. However, to do this you need to install `istioctl` to interact with Istio.
+
+The articles in this [series](https://yq.aliyun.com/articles/599874) provide detailed instructions about how to use Istio on AliCloud Kubernetes Container Service.
+
+## Uninstalling
+
+1. Navigate to the Release section of the AliCloud Container Service console at [https://cs.console.aliyun.com/#/k8s/release/list](https://cs.console.aliyun.com/#/k8s/release/list)
```

Comment:
1. Visit [the Release section of the AliCloud Container Service console](https://cs.console.aliyun.com/#/k8s/release/list).